capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -1,7 +1,22 @@
 package com.google.univiz.api;
 
+import java.util.List;
+
 /**
  * The SearchResoure interface is responsible for all data management/processing specific for the
  * search feature of the UniViz webapp.
  */
-public interface SearchResource {}
+public interface SearchResource {
+  /**
+   * Given a potentially incomplete search query, returns a list of colleges that would match the
+   * query.
+   *
+   * @param query The search query as the user typed in. It should be a non-empty and a non-null
+   *     string.
+   * @return An ordered list of possible matching colleges. The list is ordered such that the best
```

Comment:
```suggestion
   * @return A list of possible matching colleges. The list is ordered such that the best
```

Lists are inherently ordered :)